### PR TITLE
big boi pr

### PR DESCRIPTION
**Pull request by NatC02**

### CODE REVIEW

File: js/superHero.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+class superHeroMovies {
+
+	isSequel = true || false;
```
Comment (carlosrymer) [inline js/superHero.js]: As mentioned, just set this to true as it will always evaluate to true.

File: js/superHero.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    isStreamed = true || false;
+
+	constructor(props) {
+		this.isSequel = props.isSequel;
```
Comment (carlosrymer) [inline js/superHero.js]: If these are always passed as props, no need to declare as public props of the class. You would reserve those for locally global variables that you want to use to store some changeable state in memory for each instance.

File: js/superHero.js (hunk 1 above)
Comment (NatC02) [reply js/superHero.js]: Edited it.

File: js/superHero.js (hunk 2 above)
Comment (NatC02) [reply js/superHero.js]: Edited it. I went out of my way to read the following below as well.

https://blog.sessionstack.com/how-javascript-works-memory-management-how-to-handle-4-common-memory-leaks-3f28b94cfbec

**Pull request merged**